pyproject.toml

```diff
@@ -76,6 +76,10 @@ include = [
   "/requirements.txt",
 ]
 
+[tool.pyright]
+include = ["gradio/**/*.py"]
+exclude = ["gradio/themes/"]
```

Comment:
Will be pretty hard to type themes given the dynamic nature of all the theme props.